src/Sylius/Bundle/CoreBundle/Resources/config/services/listeners.xml

```diff
@@ -86,5 +86,10 @@
             <argument>_sylius.cart</argument>
             <tag name="kernel.event_subscriber" />
         </service>
+        <service id="sylius.listener.order_customer_ip" class="Sylius\Bundle\CoreBundle\EventListener\OrderCustomerIpListener">
```

Comment:
Why aren't we doing it during order processing, just as with prices and everything else?